portfolio/src/main/webapp/index.html

```diff
@@ -7,11 +7,24 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+    <div class="content">
+      <h1> Alisha Kewalramani</h1>
+      <nav>
+          <ul id = "horizontal-list">
+            <li><a href= "index.html">Home </a></li>
+            <li><a href="https://www.linkedin.com/in/ali-kew">LinkedIn</a></li>
+            <li><a href="https://github.com/alikew24"> Github </a></li>
+            <li><a href="projects.html"> Projects </a></li>
+            <li><a href= "hobbies.html">Hobbies </a></li>
```

Comment:
nit: use a consistent pattern when it comes to spaces in your HTML; most common is to omit spaces around `=` and between tags/content